src/input_formatter/text_formatter.test.tsx

```diff
@@ -91,10 +91,17 @@ test("replace detected expression-elements into readable prolog commands", () =>
 	expect(result).toEqual(expectedResult);
 });
 
+test("expression formatter testcase", () => {
+	emptyIssueList();
+	const result: string = textFormatter("Just another test $a .and a$.. test $test2$$test3$");
+	const expectedResult = "[[just,another,test],[a .,and,a],[test],[test2],[test3]].";
```

Comment:
If I understand this correctly the dot will only appear in the list if it put inside of expressions marked by `$...$`?

Reply:
yes, like it should be.